source/configuration_manual/sieve/managesieve/client_issues.rst

```diff
@@ -0,0 +1,71 @@
+.. _sieve_managesieve-client_issues:
+
+ManageSieve Client Issues
+=========================
+
+Although this ManageSieve server should comply with the draft
+specification of the ManageSieve protocol, quite a few clients don't.
+This page lists the known client problems.
+
+The TLS problem
+---------------
+
+-  The core of the TLS problem is that a ManageSieve server is required
+   to send an unsolicited CAPABILITY response right after successful TLS
+   negotiation. Older Cyrus servers did not do this and many clients
+   incorporated this protocol error as the standard, meaning that these
+   do not expect the CAPABILITY response and thus fail with subsequent
+   commands. However, now that Cyrus' Timsieved has changed its
+   behaviour towards protocol compliance, all those clients will follow
+   eventually. The following clients are known to have this TLS issue:
+
+   Thunderbird Sieve add-on
+      TLS broken for old versions. Starting with version 0.1.5 the
+      Thunderbird Sieve add-on properly supports TLS.
+
+   KMail + kio_sieve
+      TLS broken for old versions. This issue is fixed at least in kmail
+      1.9.9 / kde 3.5.9.
+
+   SquirrelMail/AvelSieve
+      For some users the !Avelsieve client stores scripts but fails to
+      retrieve them later. This problem is directly caused by
+      AvelSieve's TLS support. A quick way to fix this is not to enable
+      TLS for ManageSieve. AvelSieve stable (v1.0.1) does not have TLS
+      support at all, so you will see this happen only with the
+      development or SVN versions. Another issue is that (at least with
+      avelsieve-1.9.7) it is impossible to delete the last rule of a
+      script. For avelsieve-1.9.7 you find a patch that fixes these two
+      issues
+      `here <http://pigeonhole.dovecot.org/client-patches/avelsieve-1.9.7-dovecot.patch>`__.
```

Comment:
not sure what you mean.

Reply:
In line 13 there is a leading `-` character, that makes everything that is appropriately indented a list item. This is not necessary, as there is only one paragraph. On the other hand the client list with TLS issues are only indented items in source, not in render.

So essentially:
- Remove the `-` on line 13,
- unindent the leading text paragraph, and
- prepend "Thunderbird Sieve add-on", "KMail + kio_sieve" and "SquirrelMail/AvelSieve" with list item prefix characters (I guess it's `-`). But make sure the description under the client names is still indented, so that it does not render in one line.

Reply:
this is supposed to be a [definition list](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#definition-lists).